app/src/models/repository.ts

```diff
@@ -67,3 +67,13 @@ export interface ILocalRepositoryState {
    */
   readonly changedFilesCount: number
 }
+
+/**
+ * Returns the full name `org/repo_name` if possible otherwise the
```

Comment:
:art: to be consistent with the actual fields used, can we use `owner/name` here? Also, a suggested tweak for the first bit:

> Returns the `owner/name` alias if associated with a GitHub repository, otherwise the ...